packages/terra-form-input/src/Input.scss

```diff
@@ -0,0 +1,92 @@
+:local {
+  .form-input {
+    background-color: var(--terra-form-input-background-color, #fff);
+    background-image: var(--terra-form-input-focus-animation-background, none);
```

Comment:
For styles which are needing for theming, but are not needed in the default theme, you can remove the fallback value. For example, you can update `background-image: var(--terra-form-input-focus-animation-background, none);` to `background-image: var(--terra-form-input-focus-animation-background);`.

It looks like the following properties can drop the fallback value:
* background-image
* background-size
* box-shadow
* color (possibly, need to test)
* opacity
* transition-duration (possibly, need to test)
* transition-timing-function (possibly, need to test)

Reply:
Done d9b680c536c0806fb22c71bc5f8ea3265244136b